Added new questions (#27)
ChiperSoft/InterviewThis: README.md

  - Will it have an SSD and as much ram as can fit?
  - How hard do I have to justify software purchases?
  - How often will I receive hardware upgrades?

>   - What is the Policy if laptop get break/corrupted?

Reviewer: ```suggestion
  - What is the Policy if a laptop is broken or corrupted?
```

  - Are developers expected to do it?
  - Are there graphics designers in the company?
  - Are there UX designers in the company?

>   - Are Full Stack Developers responsible to design User Interface?

Reviewer: This is the same question as the three above it.
